<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2D Image</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item row="5" column="2" colspan="5">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Limits</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>5</number>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <item row="2" column="1">
        <widget class="QLabel" name="x_lab">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>y axis</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="value_x_min"/>
       </item>
       <item row="0" column="2" colspan="4">
        <widget class="QComboBox" name="combodata_xcoord">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>min/max:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QToolButton" name="button_flip_y">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 0px</string>
         </property>
         <property name="text">
          <string>⇄</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>min/max:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QToolButton" name="bool_log_y">
         <property name="text">
          <string>log</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="value_x_max"/>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="value_y_min"/>
       </item>
       <item row="1" column="5">
        <widget class="QToolButton" name="bool_log_x">
         <property name="text">
          <string>log</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="4">
        <widget class="QComboBox" name="combodata_ycoord">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="button_flip_x">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 0px</string>
         </property>
         <property name="text">
          <string>⇄</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="x_lab">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>x axis</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLineEdit" name="value_y_max"/>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>pixels:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="4">
        <widget class="QComboBox" name="combodata_aspect"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Slicing</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
